modules/ztp-prerequisites-for-deploying-the-ztp-pipeline.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-prerequisites-for-deploying-the-ztp-pipeline_{context}"]
+= Prerequisites for deploying the ZTP pipeline
+
+* Openshift cluster version 4.8 or above and Red Hat GitOps Operator is installed.
+* {rh-rhacm-first} version 2.3 or above is installed.
+* For disconnected environments, make sure your source data Git repository and `ztp-site-generator` container image are accessible from the hub cluster.
+* If you want additional custom content, such as extra install manifests or custom CRs for policies, add them to the `/usr/src/hook/ztp/source-crs/extra-manifest/` directory. Similarly, you can add additional configuration custom resources, as referenced from a PolicyGenTemplate, to the `/usr/src/hook/ztp/source-crs/` directory.
```

Comment:
s/custom CRs/custom resources (CR)
s/configuration custom resources/configuration CRs
s/PolicyGenTemplate/`PolicyGenTemplate`

Reply:
Should this bullet be listed as a prereq or a procedure?

Reply:
changes made.